src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_attributes.html.twig

```diff
@@ -0,0 +1,30 @@
+<div id="attributes">
+    <h4 class="ui top attached inverted black header">{{ 'sylius.ui.attributes'|trans }}</h4>
+    <div class="ui attached segment">
+        <div class="ui styled fluid accordion">
+            <div class="ui top attached tabular menu">
+                {% for locale in sylius_available_locales() %}
```

Comment:
Since we iterate every attribute, we can calculate which locales need to be shown here (and there won't be any empty ones then). This would remove the need for a custom Twig function.